tests/rptest/tests/partition_movement.py

```diff
@@ -103,8 +103,16 @@ def keep(p):
                 result.append(dict(node_id=node_id, core=partition["core"]))
         return result
 
-    def _wait_post_move(self, topic, partition, assignments, timeout_sec):
+    def _wait_post_move(self,
+                        topic,
+                        partition,
+                        assignments,
+                        timeout_sec,
+                        previous_assignmets=None):
         admin = Admin(self.redpanda)
+        admin.wait_stable_configuration(topic=topic,
```

Comment:
`wait_stable_configuration` returns configuration so it may be enough just to check it

Reply:
Move can be not started when we start checking it.
So for most cases I prefer to wait for desired state, not stable configuration

Reply:
that's good but in the case of cancellation of A->B: A and B both are desired configuration; it we can't make it nice with common utility method - we need to introduce one specific for that case. Duplication is better that wrong abstraction.

In case of ht_partition_movement_test we know that it started because `_request_move_cancel` checks it first and cancel it.